templates/redhat/dhcpd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=DHCPv4 Server Daemon
+Documentation=man:dhcpd(8) man:dhcpd.conf(5)
+Wants=network-online.target
+After=network-online.target
+After=time-sync.target
+
+[Service]
+Type=notify
+ExecStart=/usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid  <%= @dhcp_interfaces.join(' ') %>
```

Comment:
But I must define ExexStart twice  when I add ExecStart  in execstart.conf , or the service will be error. It looks very ugly.
i.e：
ExecStart=
ExecStart=/usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid br0